lib/routes/charge/items/item_page.dart

```diff
@@ -262,6 +296,9 @@ class ItemPageState extends State<ItemPage> {
               } else {
                 AddItem addItem = AddItem(
                   Item(
+                      imageURL: _itemImage != null && _itemImage.isNotEmpty
```

Comment:
why not just _itemImage as in the update case above?

Reply:
Update case was faulty and saved null value as empty string.
[Fixed](https://github.com/breez/breezmobile/pull/319/commits/257a6f0237bc1914343dc19e3cd22418c1af1367)